packages/office-ui-fabric-react/src/components/Label/LabelCodepenButton.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { CommandButton } from '../..';
+import { css } from 'office-ui-fabric-react/lib/Utilities';
+
+const jsContent = require('!raw-loader!office-ui-fabric-react/src/components/Label/codepen_examples/Label.Basic.CodepenExample.txt') as string;
+const htmlContent = `<script src="//unpkg.com/office-ui-fabric-react/dist/office-ui-fabric-react.min.js"></script>
```

Comment:
this will be the same for ALL codepen examples. we want 1 place to change this if necessary